src/main/webapp/app/overview/course-exercises/course-exercise-row.component.ts

```diff
@@ -157,6 +165,13 @@ export class CourseExerciseRowComponent implements OnInit, OnDestroy {
         return participation.results && participation.results.length > 0;
     }
 
+    isExerciseInDuedate(exercise: Exercise): boolean {
+        if (exercise.dueDate) {
```

Comment:
Optional: you can do a one liner here: `return exercise.dueDate ? exercise.dueDate.isAfter(moment()) : true;`

Reply:
Of course, I think I need some more experience until I see such things directly ;-)